### ISSUE TITLE
Graphics crash on startup

### ISSUE THREAD

**Issue by Tom-III** [godotengine/godot#12920]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Macbook Air 2012 Godot 3.0 Alpha 2
Intel HD Graphics 3000 Latest 2015 driver.

**Issue description:**

Crash on startup
![Godot error](https://i.imgur.com/2cvhZdEg.png)
**Steps to reproduce:**
Launch alpha 2

**Link to minimal example project:**

**Comment by reduz** [comment]
You dont have OpenGL support in your computer.

Lets use this issue to remind that we should show a warning when GLAD cant
initialize.

**Comment by Tom-III** [comment]
But I do have openGL, minecraft which is opengl based works just fine

**Comment by akien-mga** [comment]
You need OpenGL Core Profile 3.3, and it's likely not supported by the old macOS drivers for Intel HD 3000. (macOS OpenGL support was notoriously bad in those days).

**Comment by Tom-III** [comment]
I read that godot 2.1 will be supported until GLES2 has been updated for godot3?  Please make it compatible with gpus/computers that don't have OpenGL Core Profile 3.3

**Comment by akien-mga** [comment]
Well the above issue is about Godot 3.0 which requires OpenGL ES 3.0 / OpenGL 3.3.
Have you tried Godot 2.1.x? It should work fine on that hardware, it requires only OpenGL 2.1.

**Comment by Tom-III** [comment]
Yes, it works great however I am interested in the CSharp features of Godot 3.  This is why I wanted to try out the new builds

**Comment by akien-mga** [comment]
You will have to wait for Godot 3.1 to run it on this hardware then; or install Linux in dual boot, as the MESA drivers for Intel HD 3000 should support GL 3.3.

**Comment by Tom-III** [comment]
Ehh I'll wait I don't like linux

**Comment by reduz** [comment]
kicking to 3.1, so karroffel can finish his GLES2 backend.
